packages/react-components/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -30,6 +30,15 @@ const useStyles = makeStyles({
     flexBasis: '700px',
     flexGrow: 1,
   },
+  argsTable: {
+    // hide editors in ArgsTable
+    '& .docblock-argstable-body>tr>td:nth-child(4)': {
```

Comment:
Is it possible to use this in `preview.js` in any way ? https://storybook.js.org/docs/react/essentials/controls#filtering-controls

This kind of CSS hack seems really shaky during SB upgrades and the person who upgraded might have reenabled controls (or worse) without knowing why.

If we really wanted to keep these controls hidden, we sould do it from the `.storybook/preview-head-template.html` file where the other overrides are

Reply:
I have checked docs, discussions and source code and unfortunately have not find a clean way.
I see the following options:
- live with the editable args table (confusing, not usable)
- implement the read only args table from scratch (non trivial amount of work, similar issues with SB upgrades)
- go with the CSS hack (this pr) and add some kind of protection

I am not sure it would make sense to implement any tests.
What about having just a manually crafted storybook upgrade checklist?

Example:

> #20695 ArgsTable has prop edit column and reset props button hidden
> **Before**
> ![image](https://user-images.githubusercontent.com/9615899/144045244-ff9586cd-2ef6-46d3-bd33-3a21397ab68d.png)
> **After**
> ![image](https://user-images.githubusercontent.com/9615899/144045493-ba30a62b-defe-4f7e-80d5-7882ed1f9309.png)

(or the checklist can contain only the list with the before/after being the part of the PR)

Reply:
@miroslavstastny  in that case, as i said please put these overrides in ` .storybook/preview-head-template.html`  so that we have all our fluent specific SB overrides in the same place

` ` `css
.docblock-argstable-body>tr>td:nth-child(4) {
    display: none;
  }
  .docblock-argstable-head>tr>th:nth-child(4) {
    display: none;
  }
` ` `